src/ui/components/tenants/tenantDetail/LoginMethodsSection.tsx

```diff
@@ -159,29 +157,33 @@ const LoginFactor = ({
 					} else {
 						throw new Error(res.status);
 					}
+				} else {
+					setError(null);
```

Comment:
The logic comes here when the status is OK. This is there to reset the error in case there was an error with the factor previously but now when toggling again the error is no longer there so we reset the error. We only show error for the factor if error is non null

Reply:
This will mostly happen in the local dev, when user tries to toggle, and then we get recipe not enabled error. Then user adds recipe and restarts the backend, but not refresh the dashboard page, but now tries to turn it on. We then want the error sign to go away.